cmd/migrate/queries/queries.go

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+package queries
```

Comment:
Thinking that it rather makes sense to split the functions here into two files:
- `cmd/migrate/mariadb/queries.go`
- `cmd/migrate/sqlite/queries.go`

and then the queries go under their respective `package`s where they will be used. Thoughts?